teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(id, age, name, parent_permission, specialization)
+    super(id, age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Please do the same as I mentioned above and it would be nice if you test your `class` functionality by creating the instance and accessing it`s method to verify the results.